articles/user-guide/transmission-best-practices.md

```diff
@@ -47,12 +47,16 @@ If you receive a 504 after making a transmission request, you can inspect event
 
 For other network timeouts where it is unclear if your request reached the server, you can also monitor and retry.
 
+**Transmission API Latency**
+
+It is recommended that you set a timeout interval of 5 minutes for retrying messages if you receive no response from the transmission API. 
+
 **Monitoring for Errors**
 
 There are two ways to track transmission errors:
 
 * The response to your original transmission request.
-* Data sent to your webhook endpoint (more information on our webhooks can be found here). 
+* Data sent to your webhook endpoint (more information on our webhooks can be found [here](https://www.sparkpost.com/docs/tech-resources/extended-error-codes/)). 
```

Comment:
The attached link https://www.sparkpost.com/docs/tech-resources/extended-error-codes/ are for the errors returned in the API call itself. 

When the transmission fails to generate, we log a gen_fail event which looks something like this 
{
	"msys": {
		"gen_event": {
			"rcpt_to": "johara2661@q.com",
			"reason": "554 5.3.3 [internal] Error while rendering part text/plain: line 5: substitution value 'STATE' did not exist or was null",
			"event_id": "30880492241542103",
			"rcpt_subs": {},
			"message_id": "000105846b5b433ee973",
			"gen_successful_count": "1",
			"sending_ip": "35.163.237.243",
			"template_id": "template_30880492233740476",
			"binding": "default",
			"binding_group": "default",
			"type": "gen_fail",
			"routing_domain": "q.com",
			"timestamp": "1533772805",
			"rcpt_tags": [],
			"template_version": "0",
			"customer_id": "35984",
			"open_tracking": "1",
			"error_code": "554",
			"tenant_id": "spc",
			"ip_pool": "shared",
			"node_name": "a.mta1vrest.cc.aws-usw2a.prd.sparkpost",
			"click_tracking": "1",
			"initial_pixel": "1",
			"rcpt_meta": {},
			"raw_reason": "554 5.3.3 [internal] Error while rendering part text/plain: line 5: substitution value 'STATE' did not exist or was null",
			"recv_method": "rest",
			"transmission_id": "30880492233740476",
			"gen_rejection_count": "0",
			"ip_pool_raw": "general_3"
		}
	}
}

Reply:
afaict `gen_fail` is internal, this event would be surfaced as `generation_failure`.  Anyways this entire line should just be:
```
* [Webhooks](https://developres.sparkpost.com/api/webhooks)
```